internal/controller/csiaddons/persistentvolumeclaim_controller.go

```diff
@@ -258,7 +267,7 @@ func (r *PersistentVolumeClaimReconciler) determineScheduleAndRequeue(
 	driverName string,
 ) (string, error) {
 	annotations := pvc.GetAnnotations()
-	schedule, scheduleFound := getScheduleFromAnnotation(logger, annotations)
+	schedule, scheduleFound := getScheduleFromAnnotation(rsCronJobScheduleTimeAnnotation, logger, annotations)
```

Comment:
pass the annotation as part of `determineScheduleAndRequeue function argument so the caller can pass the required key to check